### PR TITLE
Add possibility to change UID:GID of backup file

### PR DESCRIPTION
**Pull request by tekgator**
Currently backup files are created with file user/group ownership of the root user. 

A change has been made so the user can provide a UID:GID in the container environment. When the file has been moved to the archive location the ownership is changed to the provided UID:GID.

Also added test case under `test`.

### CODE REVIEW

**Comment by varhub** [comment]
Seems great for simple '-v' cases. But, is `-R` required at chown?

**Comment by tekgator** [comment]
@varhub The `-R` is a copy paste error. And you're right, replaced `echo` statement with `-v`.

File: README.md [hunk 1]
```diff
@@ -176,6 +176,8 @@ Variable | Default | Notes
 `BACKUP_CRON_EXPRESSION` | `@daily` | Standard debian-flavored `cron` expression for when the backup should run. Use e.g. `0 4 * * *` to back up at 4 AM every night. See the [man page](http://man7.org/linux/man-pages/man8/cron.8.html) or [crontab.guru](https://crontab.guru/) for more.
 `BACKUP_FILENAME` | `backup-%Y-%m-%dT%H-%M-%S.tar.gz` | File name template for the backup file. Is passed through `date` for formatting. See the [man page](http://man7.org/linux/man-pages/man1/date.1.html) for more.
 `BACKUP_ARCHIVE` | `/archive` | When this path is available within the container (i.e. you've mounted a Docker volume there), a finished backup file will get archived there after each run.
+`BACKUP_UID` | `root (0)` | After backup file has been moved to archive location the file user ownership is changed to this UID.
+`BACKUP_GID` | If not provided `BACKUP_UID` is used | After backup file has been moved to archive location the file group ownership is changed to this GID.
```
Comment (jareware) [inline README.md:180]:
```suggestion
`BACKUP_GID` | `$BACKUP_UID` | After backup file has been moved to archive location the file group ownership is changed to this GID.
```

**Review by jareware — APPROVED** [review]
This is great, thank you for the contribution!

Rolling out a release soon.

**Pull request merged**